launch/state_machines/challenge_carry_my_luggage.launch

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<launch>
+    <arg name="robot_name" value="hero" />
+
+    <remap from="/diagnostics" to="/$(arg robot_name)/diagnostics" />
```

Comment:
Why is this needed? This is undesired.

Reply:
tradition. It is present in the other challenge files as well.

Reply:
Ok, I see it is indeed there and needed. https://github.com/tue-robotics/hero_bringup/commit/bfaa745f07b7d36829db4718f7b546656f0fa2ef